draft-packages/table/KaizenDraft/Table/Table.module.scss

```diff
@@ -72,8 +68,6 @@ $row-height-data-variant: 48px;
 // The .headerRowCell wrapper is required to increase the specificity, as so
 // we can override the `Tooltip` component's styling.
 .headerRowCell .headerRowCellTooltip {
```

Comment:
Have you had a look at changing the [Tooltip display prop](https://cultureamp.design/storybook/?path=/docs/components-tooltip--docs)? Might remove the need to apply any extra styling.

Reply:
Thanks @gyfchong, I tried but it doesn't seem to help. The problem is with the header row expanding the full width.